src/components/SchemaTable/index.jsx

```diff
@@ -1,10 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types'; // eslint-disable-line no-unused-vars
+import './index.css'; // temporary stylesheet for testing (TODO: refactor to use MUI styles)
+
+// TODO: eliminate after testing
+const exampleSchema = require('../../../schemas/basicDataTypes/string/stringPattern.json');
```

Comment:
We can avoid introducing this here by adding this example in the styleguide. The styleguide is a good way to test things there :)

Reply:
Awesome! 🚀  Will be using the styleguide to test examples from now on.